plugins/amazonq/shared/jetbrains-community/resources/software/aws/toolkits/resources/AmazonQBundle.properties

```diff
@@ -1,4 +1,7 @@
 action.q.hello.description=Hello description
+amazonqChat.incompatible.text=JCEF, a dependency of Amazon Q chat is not supported in {0}
```

Comment:
```suggestion
amazonqChat.incompatible.text=JCEF, a dependency of Amazon Q Chat is not supported in {0}
```

Reply:
maybe too long? break into 2 lines?

"Webview support is not available"
"{amazonqChat.incompatible.text}"